src/ucp/core/ucp_context.c

```diff
@@ -359,6 +365,11 @@ static ucs_config_field_t ucp_config_table[] = {
    "lane without waiting for remote completion.",
    ucs_offsetof(ucp_config_t, ctx.rndv_put_force_flush), UCS_CONFIG_TYPE_BOOL},
 
+  {"SA_DATA_VERSION", "v1",
+   "Defines UCP sockaddr private data format (which is used in client-server\n"
+   "connection establishment flow)",
```

Comment:
need to somehow document/name that this is the version that the **client sends by default**
since:
- client can send newer version when using specific APIs
- server can parse both versions
maybe ctx.sa_client_min_hdr_version ?